Oqtane.Client/Modules/Admin/Pages/Index.razor

```diff
@@ -3,23 +3,24 @@
 @inject NavigationManager NavigationManager
 @inject IPageService PageService
 
-@if (PageState.Pages != null)
-{
-    <ActionLink Action="Add" Text="Add Page" />
-
-    <Pager Items="@PageState.Pages.Where(item => !item.IsDeleted)">
-        <Header>
-            <th>&nbsp;</th>
-            <th>&nbsp;</th>
-            <th>Name</th>
-        </Header>
-        <Row>
-            <td><ActionLink Action="Edit" Parameters="@($"id=" + context.PageId.ToString())" /></td>
-            <td><ActionDialog Header="Delete Page" Message="@("Are You Sure You Wish To Delete The " + context.Name + " Page?")" Action="Delete" Security="SecurityAccessLevel.Admin" Class="btn btn-danger" OnClick="@(async () => await DeletePage(context))" /></td>
-            <td>@(new string('-', context.Level * 2))@(context.Name)</td>
-        </Row>
-    </Pager>
-}
+<div class="container">
```

Comment:
app css class selector could be included here and possibly even one in the pane div which loads prior to this one.  this has really no effect on the UX currently however was a work around for getting max width for the content pane.